src/scripts/crafttweaker/postInit.zs

```diff
@@ -292,6 +292,7 @@ function initIntegrations() {
 	scripts.crafttweaker.integrations.mods.betterwithmods.init();
 	scripts.crafttweaker.integrations.mods.bloodmagic.init.init();
 	scripts.crafttweaker.integrations.mods.chisel.init();
+	// scripts.crafttweaker.integrations.mods.cyclicmagic.init(); // TODO: uncomment this line when Cyclic's CT integration is fixed
```

Comment:
What exactly is wrong with the integration?

Reply:
Lothrazar/Cyclic#1100

Removing a recipe works, but causes a concurrent modification exception, which terminates CraftTweaker execution.